### PR TITLE
Update polyfill imports

### LINKED ISSUE #1460: @shopify/polyfills breaks SSR
## Overview

With `@shopify/polyfills` >= `1.1.9` the following happens:

```

Test suite failed to run
--
  |  
  | Cannot find module 'core-js/stable' from 'base.js'
  |  
  | Require stack:
  | node_modules/@shopify/polyfills/dist/src/base.js
  | app/foundation/App/App.tsx
  | app/foundation/App/index.tsx
  | app/index.tsx
  | server/middleware/auth-redirect-loop/auth-loop-response.tsx
  | server/middleware/auth-redirect-loop/auth-redirect-loop.ts
  | server/middleware/auth-redirect-loop/index.ts
  | server/middleware/index.ts
  | server/tests/utilities/create-context.ts
  | server/tests/utilities/index.ts
  | server/middleware/load-testing/tests/load-testing.test.ts
  |  
  | at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:299:11)
  | at Object.<anonymous> (node_modules/@shopify/polyfills/dist/src/base.js:1:1)

```
...

## Consuming repo

> What repo were you working in when this issue occurred?

Shopify/web

## Labels

- [x] Add the `Type: Bug` label to this issue.

### Area

- [x] Add any relevant `Area: <area>` labels to this issue

### Scope

- Is this issue related to a specific package?

  - [ ] Tag it with the `Package: <package_name>` label.

---

## Checklist

- [ ] Please delete the labels section before submitting your issue
- [ ] I have described this issue in a way that is actionable (if possible)

### PR DESCRIPTION
**Pull request by vsumner**
## Description

Fixes #1460 

## Type of change

- [ ]  Patch: Bug/ Documentation fix (non-breaking change which fixes an issue or adds documentation)
- [x]  Minor: New feature (non-breaking change which adds functionality)
- [ ]  Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have added a changelog entry, prefixed by the type of change noted above

### CODE REVIEW

File: packages/polyfills/src/base.ts [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+require('core-js/fn/array/includes');
+require('core-js/fn/string/pad-start');
+require('core-js/fn/string/pad-end');
+require('core-js/fn/symbol/async-iterator');
+require('core-js/fn/object/get-own-property-descriptors');
+require('core-js/fn/object/values');
+require('core-js/fn/object/entries');
+require('core-js/fn/promise/finally');
+// Ensure that we polyfill ES6 compat for anything web-related, if it exists.
+require('core-js/web');
```
Comment (alexandcote) [inline packages/polyfills/src/base.ts:13]: We still need the `require('regenerator-runtime/runtime');`

**Comment by vsumner** [comment]
🎩 looks good.

**Comment by brendo** [comment]
Hmm, were we expecting this to add ~20kb to [our entrypoints](https://shrink-ray.shopifycloud.com/repos/brochure2/commits/b6a838e9800d1f32c9ff71c7cd19b43e96d319a6/entrypoints)?

**Comment by alexandcote** [comment]
@brendo no this is an issue.

**Comment by alexandcote** [comment]
After some investigation, 

### With @babel/polyfills
image

### Without @babel/polyfills
image

We should revert this PR: https://github.com/Shopify/quilt/pull/1436

**Comment by brendo** [comment]
Thanks for the quick response Alexandre!

**Pull request merged**